imu_angular_velocity_cov:
- - 7.203030395307532e-07
  - -4.0920539851374006e-08
  - -1.018128140779069e-08
- - -4.0920539851374006e-08
  - 8.460545825896745e-07
  - -3.8630516714366413e-08
- - -1.018128140779069e-08
  - -3.8630516714366413e-08
  - 8.958797302126714e-07
imu_angular_velocity_mean:
- -0.05335163802281022
- 0.0022072279527783393
- 0.01665967035666108
imu_linear_acceleration_cov:
- - 0.00013270191773343435
  - 1.5578969017469275e-05
  - 2.644267826171487e-06
- - 1.5578969017469275e-05
  - 0.00013977236714945548
  - -3.6670450990694144e-06
- - 2.644267826171487e-06
  - -3.6670450990694144e-06
  - 0.0002722358827369759
imu_linear_acceleration_mean:
- -0.38458061185479164
- -0.3717812117636204
- -10.310705743789672
